src/js/services/getImgService.js

```diff
@@ -4,44 +4,43 @@
         .module('flickrPOC')
         .service('images', images);
 
-        function images($http){
-            return {
-                loadPics: loadPics,
-                createImagesArray: createImagesArray
-            };
-
-            function loadPics() {
-                return $http.get(_formUrl());
-                function _formUrl(){
-                    var api_key = '74e47a159e15cbcb6139ba9c9df64c13';
-                    var method = "flickr.photos.search";
-                    var user_id = "138698049@N03";
-                    var format = '&format=json';
-
-                    return "https://api.flickr.com/services/rest" + "?method=" + method + "&api_key=" + api_key + "&user_id=" + user_id + format + '&nojsoncallback=1';
-                }
+    function images($http){
+        return {
+            loadPics: loadPics,
+            createImagesArray: createImagesArray
+        };
+
+        function loadPics() {
+            return $http.get(_formUrl());
+            function _formUrl(){
+                var api_key = '74e47a159e15cbcb6139ba9c9df64c13';
+                var method = "flickr.photos.search";
+                var user_id = "138698049@N03";
+                var format = '&format=json';
+
+                return "https://api.flickr.com/services/rest" + "?method=" + method + "&api_key=" + api_key + "&user_id=" + user_id + format + '&nojsoncallback=1';
             }
+        }
 
-            function createImagesArray(data){
-                var farm, server, id, title, secret, picUrl, currentPhoto;
-                var images = [];
-
-                for(var i = 0; i < data.photos.photo.length; i++) {
-                    currentPhoto = data.photos.photo[i];
+        function createImagesArray(data){
+            var farm, server, id, title, secret, picUrl, currentPhoto;
+            var images = [];
 
-                    farm = currentPhoto.farm;
-                    server = currentPhoto.server;
-                    id = currentPhoto.id;
-                    title = currentPhoto.title;
-                    secret = currentPhoto.secret;
-                    picUrl = "https://farm" + farm + ".staticflickr.com/" + server + "/" + id + "_" + secret + "_z.jpg";
+            for(var i = 0; i < data.photos.photo.length; i++) {
+                currentPhoto = data.photos.photo[i];
 
-                    images.push(picUrl);
-                }
+                farm = currentPhoto.farm;
+                server = currentPhoto.server;
+                id = currentPhoto.id;
+                title = currentPhoto.title;
+                secret = currentPhoto.secret;
+                picUrl = "https://farm" + farm + ".staticflickr.com/" + server + "/" + id + "_" + secret + "_z.jpg";
 
-                return images;
+                images.push(picUrl);
             }
 
+            return images;
         }
-})(angular);
 
+    }
+})(angular);

```

Comment:
Same goes for these changes. If they aren't required for the controller updates, please create a new PR for these updates.